src/coreclr/utilcode/stresslog.cpp

```diff
@@ -162,12 +165,12 @@ static LPVOID CreateMemoryMappedFile(LPWSTR logFilename, size_t maxBytesTotal)
 
         // append the string representation of the PID
         DWORD pid = GetCurrentProcessId();
-        wchar_t pidStr[20];
-        _itow(pid, pidStr, 10);
-        wcscat_s(fileName, pidStr);
+        WCHAR pidStr[20];
+        _itow_s(pid, pidStr, 20, 10);
```

Comment:
Can you use something like ARRAY_SIZE here for the 20 to eliminate the possibility of future mismatched `pidStr`? We use various names for that in the runtime, we have _countof or COUNTOF and both of them used at various places in utilcode.